src/Symfony/Component/BrowserKit/AbstractBrowser.php

```diff
@@ -428,7 +428,7 @@ protected function doRequestInProcess(object $request)
         }
 
         if (!$process->isSuccessful() || !preg_match('/^O\:\d+\:/', $process->getOutput())) {
-            throw new RuntimeException(sprintf('OUTPUT: %s ERROR OUTPUT: %s.', $process->getOutput(), $process->getErrorOutput()));
+            throw new RuntimeException(sprintf('OUTPUT: "%s" ERROR OUTPUT: "%s".', $process->getOutput(), $process->getErrorOutput()));
```

Comment:
should be reverted

Reply:
Same here, I've just applied patch that fabpot suggested. But I'll revert, since fabpot fails anyways